packages/web-components/fast-foundation/src/combobox/combobox.spec.ts

```diff
@@ -40,16 +40,6 @@ async function setup() {
 
 // TODO: Need to add tests for keyboard handling & focus management
 describe("Combobox", () => {
-    it("should include a control with a role of `combobox`", async () => {
-        const { element, connect, disconnect } = await setup();
-
-        await connect();
-
-        assert.strictEqual(element.control.getAttribute("role"), "combobox");
-
-        await disconnect();
-    });
-
```

Comment:
This is still true, yes?

Reply:
While it's technically still true, the attribute is now hard-coded in the template instead of reflecting a property binding.

Reply:
I think I’m scenarios like these the test is still valid and likely beneficial - this really ensures that the combobox role exists. Whether it’s hard coded or not shouldn’t matter; more that the combobox role is a big part of what makes it a combobox :)